### DISCUSSION TITLE
How to delete songs?

### DISCUSSION THREAD

**Discussion by lndfebfdjo** [ryohey/signal#270] [Q&A]
My "Open" menu is very messy, and I wanted to ask if there is a way to delete songs that I don't need. If there isn't, it'd be great if you could add that option

**Comment by ryohey** [comment]
Hi @lndfebfdjo ! You can now delete the song like 👇
<img width="497" alt="image" src="https://github.com/ryohey/signal/assets/5355966/3bcf1b69-df85-4cab-b368-8a664d5f1ad1">
